exercises/isogram/canonical-data.json

```diff
@@ -1,33 +1,44 @@
 {
-  "cases": [{
-    "input": "duplicates",
-    "expected": true
-  }, {
-    "input": "eleven",
-    "expected": false
-  }, {
-    "input": "subdermatoglyphic",
-    "expected": true
-  }, {
-    "input": "Alphabet",
-    "expected": false
-  }, {
-    "input": "thumbscrew-japingly",
-    "expected": true
-  }, {
-    "input": "Hjelmqvist-Gryb-Zock-Pfund-Wax",
-    "expected": true
-  }, {
-    "input": "Heizölrückstoßabdämpfung",
-    "expected": true
-  }, {
-    "input": "the quick brown fox",
-    "expected": false
-  }, {
-    "input": "Emily Jung Schwartzkopf",
-    "expected": true
-  }, {
-    "input": "éléphant",
-    "expected": false
-  }]
+  "cases": [
+    {
+      "description": "empty string has no duplicates",
+      "input": "",
+      "expected": true
+    },
+    {
+      "description": "isogram with only lower case characters",
+      "input": "duplicates",
```

Comment:
I'd prefer to use a different word here as 'duplicates' is misleading. (since there are no duplicates.)
`isogram` could be good, since `isogram` is an isogram